client/rest-high-level/src/test/java/org/elasticsearch/client/DataFrameRequestConvertersTests.java

```diff
@@ -159,8 +178,17 @@ public void testGetDataFrameTransform() {
         assertFalse(request.getParameters().containsKey("from"));
         assertFalse(request.getParameters().containsKey("size"));
 
-        getRequest.setFrom(0);
-        getRequest.setSize(10);
+        getRequest.setPageParams(new PageParams(0, null));
+        request = DataFrameRequestConverters.getDataFrameTransform(getRequest);
+        assertEquals("0", request.getParameters().get("from"));
```

Comment:
I find the following syntax more readable:
assertThat(request.getParameters(), allOf(hasEntry("from", "0"), not(hasKey("size"))));

WDYT?